pkg/configuration/configuration.go

```diff
@@ -197,6 +268,11 @@ func (ev *extendedViper) Clone() Configuration {
 		clone.AddDefaultValue(k, v)
 	}
 
+	clone.SetAutomaticEnv(ev.GetAutomaticEnv())
```

Comment:
Suggestion: based on my previous comment about the setter.
```suggestion
	if ev.GetAutomaticEnv() {
	  clone.AutomaticEnv()
	}
```